email-ballerina/Module.md

```diff
@@ -0,0 +1,181 @@
+## Overview
+
+This module contains functions to perform email operations such as sending and reading emails using the SMTP, POP3, and IMAP4 protocols.
+
+### Client
+
+This module supports the following three client types.
+
+- `email:SmtpClient` - The client, which supports sending an email using the SMTP protocol.
+- `email:PopClient` - The client, which supports receiving an email using the POP3 protocol.
+- `email:ImapClient` - The client, which supports receiving an email using the IMAP4 protocol.
+
+#### SMTP Client
+
+To send an email using the SMTP protocol, you must first create an `email:SmtpClient` object. The code for creating an `email:SmtpClient` can be found
+ below.
+
+##### Creating a client
+
+The following code creates an SMTP client, which connects to the default port(465) and enables SSL.
+```ballerina
+email:SmtpClient smtpClient = new ("smtp.email.com",
+                                   "sender@email.com",
+                                   "pass123");
+```
+The port number of the server can be configured by passing the following configurations.
+
+```ballerina
+email:SmtpConfiguration smtpConfig = {
+    port: 465 // Can use ports, 465, 587 or 25
+};
+
+email:SmtpClient smtpClient = new ("smtp.email.com",
```

Comment:
Is this correct ? I thought init could return error ?

Reply:
Yes. Added the error.